recipes-kernel/linux/linux-fslc-imx_5.10.bb

```diff
@@ -87,4 +87,4 @@ LOCALVERSION = "-5.10.52-2.1.0"
 
 DEFAULT_PREFERENCE = "1"
 
-COMPATIBLE_MACHINE = "(mx6|mx7|mx8)"
```

Comment:
Why `linux-fslc-imx` is compatible only with `*-nxp-bsp`?

Reply:
Do you see a reason to use linux-fslc-imx with mainline BSP?

Reply:
I don't, but if I understand it right, it was possible to use it before and now it is not possible any more.
Anyway, nothing that a future PR would not add back

Reply:
It wasn't as `use-mainline-bsp` was not accepted. So I just kept as before.

Reply:
ok